src/main/java/Vibes.java

```diff
@@ -0,0 +1,99 @@
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class Vibes {
+    private static final String CHATBOT_NAME = "Vibes";
+    private static final String DASHED_LINE = "\t---------------------------------------------------------------------------------------";
+
+    public static String extractCommand(String userInput) {
+        if (userInput.contentEquals("list")) {
+            return "list";
+        } else if (userInput.contentEquals("bye")) {
+            return "bye";
+        } else if (userInput.startsWith("mark ")) {
+            return "mark";
+        } else if (userInput.startsWith("unmark ")) {
+            return "unmark";
+        } else if (userInput.startsWith("todo") || userInput.startsWith("deadline ") || userInput.startsWith("event ")) {
+            return "add task";
+        } else if (userInput.startsWith("delete ")) {
+            return "delete";
+        } else {
+            return userInput;
+        }
+    }
+
+    private static void executeCommand(String commandToExecute, List taskList, String userInput) throws CommandNotFoundException {
```

Comment:
Can consider having a separate parser class for parsing input in the future like in personsbook